src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -105,32 +105,54 @@ public void allowRenewalLoanByIdWhenFirstRequestInQueueIsHold() throws Interrupt
   }
 
   @Test
-  public void forbidOverrideRenewalLoanByBarcodeWhenFirstRequestInQueueIsRecall() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+  public void allowOverrideRenewalLoanByBarcodeWhenFirstRequestInQueueIsHold() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
```

Comment:
This test seems to suggest that renewal override is allowed for either hold or recall request. Is that the case?

Reply:
Hmm, it's stage... I don't remember changing this test.
![image](https://user-images.githubusercontent.com/42538529/59766457-e83d4000-92a8-11e9-9e1e-f3e71fc11c02.png)
But you're right, renewal override should be allowed only in case of open recall request.
I'll change it.

Reply:
Actually it just should be removed, as there is `forbidRenewalOverrideWhenFirstRequestIsNotRecall` test.

Reply:
Done